index.js

```diff
@@ -159,7 +176,7 @@ class Replicate {
       throw new Error(`API request failed: ${response.statusText}`);
     }
 
-    return response.json();
+    return response;
```

Comment:
How come we're not returning the parsed response anymore? It looks like we parse it every time we call it anyway, and this will break anyone else using it.

Reply:
It's more helpful for this method to return an `Response` than JSON, because then we can use it to fetch SSEs with auth headers. And in the future, we may have other endpoints that have non-JSON responses (e.g. `files.get`).